Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ckan/ckan-dev:2.10.5-py3.10
+FROM --platform=linux/amd64 ckan/ckan-dev:2.10.5-py3.10
```

Comment:
Does this build on M-series Macs? I've been struggling with this separately and I don't know the best way to support multiple platform architectures in code.

Reply:
Yes, it works on M-series Macs. This flag specifies the target platform of the container (Linux in this case), not the host OS where Docker is running.

Reply:
I removed this setting to avoid complexity, as individual users can configure it in their local environment.